twoliter/embedded/build.Dockerfile

```diff
@@ -376,7 +376,7 @@ WORKDIR /tmp
 RUN --mount=target=/host \
     mkdir -p /local/archives \
     && KERNEL="$(printf "%s\n" ${PACKAGES} | awk '/^kernel-/{print $1}')" \
-    && find /host/build/rpms/ -maxdepth 2 -type f \
+    && find /host/build/ -type f \
```

Comment:
This is not ideal and needs at least an issue in the backlog to fix.

It's also only safe if we are 100% sure that older / different versions of the core kit are cleaned up. Otherwise we might copy the wrong RPM.

At a minimum we should filter the name to `bottlerocket-${KERNEL}-archive-*.${ARCH}.rpm`.

Reply:
Should I change it to the above?

Reply:
Yes, please do.